### DISCUSSION TITLE
Problem Too many redirect

### DISCUSSION THREAD

**Discussion by MrSipping** [nginx/unit#1369] [Support]
Hi I have this problem where my application makes internal redirects in the index.php file, unfortunately when I enter the Unit configuration and try to call the log in page it gives me error ‘TOO_MANY_REDIRECT’ this error via Nginx Unit I can't solve it and I wanted to know if anyone had any idea how I can do it. 

Thank you all

**Comment by lcrilly** [comment]
There is insufficient information to understand the problem.

`TOO_MANY_REDIRECT` looks like a browser error? So if your PHP code is sending redirects then this looks most likely to be an application problem.

Feel free to reopen if you think this is a problem with Unit.
